assets/base.css

```diff
@@ -2312,10 +2333,16 @@ product-info .loading-overlay:not(.hidden) ~ *,
 
   .header--top-center {
     grid-template-areas:
-      'left-icon heading icons'
+      'search heading icons'
       'navigation navigation navigation';
   }
 
+  .header--top-center.drawer-menu {
+    grid-template-areas:
+      '. search heading icons';
+    grid-template-columns: auto auto 1fr auto;
+  }
```

Comment:
There is an issue with this setup. Setting some columns to auto means that anytime the amount of content will change in those columns, it will impact the other ones and misalign the logo. 

Eg: apps in an upcoming PR and also if a store isn't enabling accounts

<details>
<summary>Screenshot</summary>

![](https://screenshot.click/21-32-bpb9z-fzhmu.png)
</details>

Reply:
You could maybe do: 
`left-icons heading icons` - `1fr auto 1fr`

Then assign both the `.header__search` & `header-drawer` to that `grid-area: left-icons;` and finally add a `margin-left: 3.2rem;` on the `header__search` which is the result of the hamburger icon width `4.4rem` minus its margin-left value `-1.2rem`

Reply:
Good catch @ludoboludo 👍.

Reply:
The change I was mentioning was just needed for that one instance I believe. I see some other grid declaration were edited and I don't believe it's needed 🤔 

I'll comment more in my upcoming review.